### PR TITLE
Siri Hemsley - Project Todo

### PR DESCRIPTION
**Pull request by sirimatsdotter**
**Link to my deployed project:** https://gracious-gates-44c449.netlify.app/

**My reflections on how this week's project turned out:**
I'm really happy with this projekt, it turned out so much better than I thought in the beginning of the week! I'm really happy that I've started to understand redux and how to create reducers and get data from the store. 

I'm happy that I could master some conditional rendering for different components and I'm really happy with the end result!

**Things I'd like to have clarified or explained in more detail:**
Nothing right now

### CODE REVIEW

**Review by Yennie-Leek — APPROVED** [review]
Really nice project Siri!💫  
Everything works well, the requierments are meet and more😀 
The code is easy to read and understand. And I like the "Woho!" image you added when no tasks to do.🥳
Well done and keep shining!😎

File: code/src/components/Main.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  flex-direction: column;
+`
+
+export const Main = () => {
+  const itemsLength = useSelector((store) => store.tasks.items.length)
+
+  return (
+    <MainWrapper>
+      <TaskForm />
+      {itemsLength === 0 ? <EmptyState /> : <TaskList />}
```
Comment (KarolCislowski) [inline code/src/components/Main.js:25]: nice solution 😸

File: code/src/components/TaskForm.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      timeStamp: timestamp
+    }
+
+    dispatch(tasks.actions.addTask(newTask))
+    setDescription('')
+  }
+
+  return (
+    <Form>
+      <SubmitButton type="submit" onClick={description !== "" ? onTaskSubmit : (e) => e.preventDefault()}><AiOutlinePlus size={20} /></SubmitButton>
```
Comment (KarolCislowski) [inline code/src/components/TaskForm.js:78]: you should spread it into 3 lines. 🤓

**Review by KarolCislowski — APPROVED** [review]
It is an awesome app 🥳

**Pull request closed without merging**